packages/file-export/fastfetchbot_file_export/video_download.py

```
@@ -0,0 +1,210 @@
+                    file_path
+                    if local_mode
+                    else base_url + "/fileDownload" + file_path
+                )
+
+        return {
+            "message": "success",
+            "content_info": content_info,
+            "orientation": orientation,
+            "file_path": file_path_output,
+        }
+    except Exception as e:
```

Ruff (0.15.1)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)